IDR/notebook/0.get_ids.ipynb

```diff
@@ -226,7 +226,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "Python 3.9.12 ('base')",
```

Comment:
Before we run the full production pipeline, let's make sure we're using a consistent python environment.

Have you used `conda` before?

There is an `environment.yml` file in this repo. Please create this environment, activate the environment, and then run the notebooks using the environment kernel.


```bash
# Make sure you're in the top level directory of this repo

# Create environment
conda env create --force --file environment.yml

# Activate environment
conda activate microscopy-data

# Run jupyter notebooks as usual (make sure to use microscopy-data kernel in vscode!)
```

Reply:
p.s. - See https://docs.conda.io/en/latest/miniconda.html for conda installation

Reply:
I have used conda before, although mostly just for installing modules rather than creating a venv. I've mostly used Docker for that. It seems that creating this conda environment seems much easier than using Docker though.

Reply:
Conda is sufficient for our purposes currently, although, depending on the directions we travel in this project, docker could be very useful! (and preferred over conda)

Reply:
It looks like the display name for the `kernel_spec` is off - It should be named what you named your `environment.yml` file.

Can you double check that vscode (or whereever you're executing your `.ipynb` notebooks) is recognizing and pulling from your specified conda env?

Reply:
I just deleted, regenerated, and renamed the conda env. It's now recognizing and suggesting (making it default) the microscopy-data environment.

Added in 6ec7b46d695b8b180e7ad196ee64274be48a3092.